website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -0,0 +1,683 @@
+- File Version (always send back latest version to compare)
+
+A rough structure may look like the following below. In order to simplify
+comparisons, only the latest file version information will be returned. Enhancements
+we will need consider later include items like updating the latest flag in the
+database and glacier on a file version when doing a recovery.
+
+```json
+[
+  {
+    "providerId": "lpdaac",
+    "collections": [
+      {
+        "collectionId": "MOD14A1__061",
+        "granules": [
+          {
+            "granuleId": "MOD14A1.061.A23V45.2020235",
+            "date": "TBD",
+            "createDate": "2020-01-01T23:00:00+0",
+            "lasstUpdate": "2020-01-01T23:00:00+0",
+            "files": [
+              {
+                "fileName": "MOD14A1.061.A23V45.2020235.2020240145621.hdf",
+                "orcaLocation": "s3://orca-archive/MOD14A1/061/...",
+                "fileSize": 100934568723,
+                "fileChecksum": "ACFH325128030192834127347",
+                "fileChecksumType": "SHA-256",
+                "fileVersion": "VXCDEG902",
+                "orcaEtag": "YXC432BGT789"
+              },
+              ...
+            ]
+          },
+          ...
+        ]
+      },
+      ...
+    ]
+  },
+  ...
+]
+```
+
+Some items we will need to think about include managing paging (start/stop) for
+large returns and limiting size of the message across the wire.
+
+
+### Potential ORCA Reconciliation Data Reporting Architecture
+
+The following provides a rough high level architecture for reporting the ORCA
+holdings to Cumulus. When providing data back to Cumulus, the solution should be
+able to handle the filters supplied and return back the needed data. In addition,
+the solution should be extensible and should allow for potential changes in the
+underlying data structure or storage engine as ORCA should be able to handle AWS
+PostgreSQL RDS, AWS Aurora PostgreSQL, and AWS Aurora PostgreSQL Serverless
+offerings.
+
+<MyImage
+    imageSource={useBaseUrl('img/Research-Reconciliation-Cumulus-Reconcile.png')}
+    imageAlt="System Context"
+    zoomInPic={useBaseUrl('img/zoom-in.svg')}
+    zoomOutPic={useBaseUrl('img/zoom-out.svg')}
+    resetPic={useBaseUrl('img/zoom-pan-reset.svg')}
+/>
+
+
+## Populating the metadata
+
+Populating the metadata objects with the proper metadata will need to come from
+various sources. The sections below will go through thoughts and ideas of where
+the metadata will originate from to populate the objects and potential ideas on
+what the underlying architecture may look like to perform the tasks.
+
+### Automated Cumulus Ingest Workflow
+
+The conventional way of populating the objects with metadata would be through
+the normal Cumulus ingest workflow that contains ORCA's `copy_to_glacier` lambda.
+The `copy_to_glacier` lambda would need to be modified to take or pull additional
+metadata needed for a granule's files like `providerId` and `collectionId` and
```

Comment:
metadata needed for a granule's files, like `providerId` and `collectionId`, and

Reply:
rewrote the sentence to make it more clear.

Reply:
Looks good.